### ISSUE TITLE
How to use sentencebert for joint learning like using simcse with other tasks

### ISSUE THREAD

**Issue by WhenHow233** [UKPLab/sentence-transformers#1548]
Hello and thank you for your work. I 'm a new NLP learner.
This is the code you provided for the fine-tuning of a pre-trained BERT model on unlabeled data using SimCSE
The code for fine tuning using SimCSE has been provided but I want to have multiple downstream tasks to learn module jointly with simcse. I don' t know how to add loss to compute. 
When I write a simcse task myself, the embedding obtained through sentenceBert is always the same.
